test/case/cli/cli-output/show-routes-ipv4.txt

```diff
@@ -6,6 +6,6 @@
    10.0.13.0/30       110/2000 e5               ospfv2    0h0m0s
 >* 10.0.13.0/30            0/0 e5               direct    0h0m0s
    10.0.23.0/30          110/1 e6               ospfv2    0h0m0s
->* 10.0.23.0/30            0/0 e6               direct    0h0m0s
-   192.168.3.0/24        110/1 e2               ospfv2    0h0m0s
->* 192.168.3.0/24          0/0 e2               direct    0h0m0s
+>* 10.0.23.0/30            0/0 e6               direct  20h10m5s
+   192.168.3.0/24        110/1 e2               ospfv2  133h5m0s
+>* 192.168.3.0/24          0/0 e2               direct  205h0m0s
```

Comment:
Might I trouble you to also update cli-pretty to convert it back to the weeks/days format as well?  
Pretty please? :pleading_face: